io/serialport/locales/de/25-serial.html

```diff
@@ -0,0 +1,106 @@
+            <li>warten bis ein Puffer mit fester Größer gefüllt ist.</li>
+        </ul>
+    </p>
+    <p>Dann wird <code>msg.payload</code> entweder als UTF-8 ASCII Zeichenfolge (string) oder
+       als binäres Buffer-Objekt ausgegeben.</p>
+    <p><code>msg.port</code> enthält den Namen der ausgewählten seriellen Schnittstelle.</p>
+    <p>Wenn kein Aufteilungszeichen vorgegeben ist oder ein Zeitlimit oder eine Puffergröße von 0 angegeben ist,
+       wird ein Strom von einzelnen Zeichen ausgegeben -
+       wieder entweder als ASCII-Zeichen oder als binärer Buffer, jeweils der Größe 1.</p>
+</script>
+
+<script type="text/html" data-help-name="serial out">
+    <p>Datenversand über lokale serielle Schnittstelle.</p>
+    <h3>Inputs</h3>
+    <dl class="message-properties">
+        <dt>payload <span class="property-type">string | buffer</span></dt>
+        <dd>Zu sendende Daten über serielle Schnittstelle</dd>
+        <dt class="optional">baudrate <span class="property-type">string</span></dt>
+        <dd>Baudrate der seriellen Schnittstelle (optional)</dd>
+    </dl>
+    <p>Nur <code>msg.payload</code> wird gesendet.</p>
+    <p>Optional kann die Baudrate geändert werden über <code>msg.baudrate</code></p>
+    <p>Optional kann das Zeilenvorschub-Zeichen (\n), was zum Aufteilen der Eingangsdaten genutzt wird, 
+       auch jeder zu sendenden Nachricht angefügt werden.</p>
+    <p>Binäre Nutzdaten (Payload) können unter Verwendung von Buffer-Objekten gesendet werden.</p>
+</script>
+
+<script type="text/html" data-help-name="serial request">
+    <p>Verbindung zu einer seriellen Schnittstelle mit Request/Response-Protokoll (Anfrage/Antwort).</p>
+    <p>Dieser Node verhält sich wie eine fest gekoppelte Kombination aus einem
+       <span style="background-color:Gainsboro">serial in</span>- und einem 
+       <span style="background-color:Gainsboro">serial out</span>-Node,
+       welche sich eine Konfiguration teilen.</p>
+    <p>Die Anfrage-Nachricht wird über <code>msg.payload</code> so versendet,
+       wie man es den <span style="background-color:Gainsboro">serial out</span>-Node machen würde.
+       Die Nachrichten werden an die serielle Schnittstelle nach dem strikten FIFO-Prinzip (First In, First Out) weitergereicht,
+       d.h. nach jeder gesendeten Anfrage wird auf die Antwort der Gegenstelle gewartet,
+       bevor mit der nächsten Anfrage fortgefahren wird.
+       Wenn die Antwort empfangen wurde (mit der selben Logik eines <span style="background-color:Gainsboro">serial in</span>-Nodes)
+       oder wenn ein Zeitablauf aufgetreten ist, wird eine Nachricht ausgegeben (siehe Ausgangsdaten unten)
+       mit <code>msg.payload</code>, die empfangene Antwort enthaltend (oder fehlend im Falle eines Zeitablaufs),
+       und allen anderen unveränderten Eigenschaften der Eingangsnachricht.</p>
+    <p>Für die Konsistenz mit dem <span style="background-color:Gainsboro">serial in</span>-Node ist 
```

Comment:
This is the Englisch original with "with the same logic of a":
       Once a response is received (with the same logic of a <code>serial in</code> node), or after a timeout occurs,
       a message is sent to the output (see Outputs below),
       with <code>msg.payload</code> containing the received response (or missing in case if timeout) and
       all other properties are preserved.</p>

Reply:
I've read the English original and still think "Analog zum serial-in-Node.." would be a good fit. "Für die Konsistenz mit" is definitely not a correct translation.